src/inbox.rs

```diff
@@ -70,6 +70,22 @@ impl<A> Chan<A> {
         self.shutdown_waiting_senders();
     }
 
+    pub fn increment_sender_count(&self) {
```

Comment:
I think I originally did this directly on TxStrong so that it wouldn't seem like any reference to the Chan is able to increment the sender count (notably, weaks cant). Though, that wasn't exactly a very robust way of encapsulating that :sweat_smile:

Reply:
I get the idea!

I'd frame `Chan` as (powerful) low-level building blocks for the high-level concepts of `Address` and `Mailbox`. Thus, I'd say we favor encapsulation over a misuse-resistant interface. The latter is nice but because it is an interface private to the crate, I think it is acceptable.

Plus, tying the decrementing and shutting down of receivers together makes it much easier to understand the internals of `xtra`.

Reply:
Fair enough!